cli/template.go

```diff
@@ -329,17 +383,26 @@ func renderTemplateCmd(c *configuration) *cobra.Command {
 	return cmd
 }
 
-func uploadTemplateCmd(c *configuration) *cobra.Command {
+func uploadTemplateCmd(cmdxConfig *cmdx.Config) *cobra.Command {
 	var fileReader = ioutil.ReadFile
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "upload",
 		Short: "Upload Templates YAML file",
 		Annotations: map[string]string{
 			"group:core": "true",
 		},
 		Args: cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			ctx := context.Background()
+			spinner := printer.Spin("")
+			defer spinner.Stop()
```

Comment:
@mabdh aprt from defer, we should also stop spinner explicitly when we have received the response from the client. otherwise, the spinner would still show even when cli is printing output.

Reply:
Yes thanks @ravisuhag for pointing that out. Will fix it.